src/app.js

```diff
@@ -3,6 +3,7 @@ import i18n from 'i18n'
 import bodyParser from 'body-parser'
 import swaggerUiExpress from "swagger-ui-express";
 import swaggerDocument from "../swagger.json";
+import userRoute from './routes/userRouters';
 
 const app = express();
 app.use(bodyParser.json());
```

Comment:
`json()` is available with newer `express`. Check to see if you really need `bodyParser` anymore.